media-libs/libcamera/libcamera-0.4.0.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 2021-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="A complex camera support library for Linux, Android, and ChromeOS"
+HOMEPAGE="https://libcamera.org/"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.libcamera.org/libcamera/libcamera.git"
+else
+	SRC_URI="https://github.com/kbingham/libcamera/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+
+#IUSE="debug doc drm gnutls gstreamer jpeg libevent qt5 sdl tiff trace udev unwind v4l2"
+IUSE="debug drm gnutls gstreamer jpeg libevent qt5 sdl tiff trace udev unwind v4l2"
+REQUIRED_USE="qt5? ( tiff )"
+
+DEPEND="
+	dev-libs/libyaml:=
+	dev-python/jinja2
+	dev-python/ply
+	dev-python/pyyaml
+	|| (
+		net-libs/gnutls
+		dev-libs/openssl
+	)
+	debug? ( dev-libs/elfutils:= )
+	gstreamer? ( media-libs/gstreamer:= )
+	libevent?
```

Comment:
Surely this isn't right if it's all purely for the demo application per metadata.xml?